### PR TITLE
CI/CD Pipeline

### LINKED ISSUE #20: Set up CI/CD Pipeline

### PR DESCRIPTION
**Pull request by sarahhtbarton**

### CODE REVIEW

File: .github/workflows/azure-kubernetes-service-helm.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
 name: Build and deploy matrix webserver app to AKS with Helm
 
 on:
-  push:
+  # push:
+  pull_request:
     branches:
       - master
```
Comment (AYM1607) [inline .github/workflows/azure-kubernetes-service-helm.yml]: Now that we know this works we should move it back to just deploy on merge to master.

File: .github/workflows/azure-kubernetes-service-helm.yml [hunk 2]
```diff
@@ -109,7 +111,6 @@ jobs:
       with:
         renderEngine: 'helm'
         helmChart: ${{ env.CHART_PATH }}
-        overrideFiles: ${{ env.CHART_OVERRIDE_PATH }}
```
Comment (AYM1607) [inline .github/workflows/azure-kubernetes-service-helm.yml:112]: Did you manage to get the override to work?

**Review by AYM1607 — CHANGES_REQUESTED** [review]

File: .github/workflows/azure-kubernetes-service-helm.yml (hunk 1 above)
Comment (sarahhtbarton) [reply .github/workflows/azure-kubernetes-service-helm.yml]: Done -- see commit "reverts trigger back to deploying when merge to master"

File: .github/workflows/azure-kubernetes-service-helm.yml (hunk 2 above)
Comment (sarahhtbarton) [reply .github/workflows/azure-kubernetes-service-helm.yml:112]: Yes! it works! See commit "attempts override values"

**Pull request merged**